### PR TITLE
Added `rgb_to_grayscale`

### PR DESCRIPTION
**Pull request by IMvision12**
@fchollet

### CODE REVIEW

File: keras/backend/torch/image.py [hunk 1]
```diff
@@ -14,6 +14,37 @@
 )
 
 
+def rgb_to_grayscale(image, data_format="channel_last"):
+    try:
+        import torchvision
+    except:
+        raise ImportError(
+            "The torchvision package is necessary to use `resize` with the "
```
Comment (fchollet) [inline keras/backend/torch/image.py:22]: `resize` -> `rgb_to_grayscale`

File: keras/ops/image.py [hunk 2]
```diff
@@ -7,6 +7,102 @@
 from keras.ops.operation_utils import compute_conv_output_shape
 
 
+class RGB_to_Grayscale(Operation):
```
Comment (fchollet) [inline keras/ops/image.py:10]: Should be `RGBToGrayscale`

File: keras/ops/image_test.py [hunk 3]
```diff
@@ -13,6 +13,11 @@
 
 
 class ImageOpsDynamicShapeTest(testing.TestCase):
```
Comment (fchollet) [inline keras/ops/image_test.py:15]: We should also test numerical correctness, in both channels_first and last format, for batched and unbatched.

**Comment by IMvision12** [comment]
Hi @fchollet Thanks for the initial review, can you tell me how can i fix failing tests?

**Comment by fchollet** [comment]
There seems to be more than one issue with the unit tests. For the particular issue you mentioned, I think you just need to make sure to call `super().__init__()`. Basically just do the same as what the other ops are doing and it will work.

**Comment by IMvision12** [comment]
@fchollet

File: keras/ops/image.py [hunk 4]
```diff
@@ -7,6 +7,103 @@
...
+            corresponds to inputs with shape `(batch, channels, height, width)`.
+            Defaults to `"channels_last"`.
+
+    Returns:
+        Grayscale image or batch of grayscale images.
+
+    Examples:
+
+    >>> import numpy as np
+    >>> from tensorflow.keras import ops
```
Comment (fchollet) [inline keras/ops/image.py:81]: `from keras import ops`

**Review by fchollet — COMMENTED** [review]
Thanks for the update -- the code looks good overall, over the tests are still failing for two backends with shape correctness issues. Please take a look.

**Pull request merged**